Documents/README.md

```diff
@@ -1,6 +1,23 @@
 
 # BeagleBoneBlack
 
+![Screenshot from 2023-09-10 11-50-32](https://github.com/PranabNandy/BeagleBone-Black-Platform-Bring-Up/assets/80820274/4e2654ec-c2b2-463d-9735-2a0e1be3d1ea)
```

Comment:
can we add it to some other readme file.

Reply:
No it is fine here